### PR TITLE
Feature/step controller

### PR DESCRIPTION
**Pull request by dominikas**
Changes with Step Controller and related things

### CODE REVIEW

File: FMM/Features/Dream/DreamController.cs [hunk 1]
```diff
@@ -1,54 +1,54 @@
-﻿using System;
-using System.Threading.Tasks;
-using FMM.Features.Dream.Commands;
-using Microsoft.AspNetCore.Mvc;
-using MediatR;
-using FMM.Features.Dream.Queries;
+﻿    using System;
```
Comment (piotrnieweglowski) [inline FMM/Features/Dream/DreamController.cs:1]: we need to revert changes in this file. We need to create a new commit and push it.
To revert changes in this particular file you can do:
git checkout develop FMM/Features/Dream/DreamController.cs 

Before doing this, please make sure that you have the recent version of develop branch on you local machine.

File: FMM/Features/Step/StepController.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
+﻿using MediatR;
```
Comment (piotrnieweglowski) [inline FMM/Features/Step/StepController.cs:1]: we need to sort it out before merging (use mediatR, remove uncommented code etc)

File: FMM/Persistent/Step.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FMM.Persistent
+{
+    public class Step
+    {
+        public Guid Id { get; set; }
+        public string Description { get; set; } 
+        public IList<Task> TaskList { get; set; }
```
Comment (piotrnieweglowski) [inline FMM/Persistent/Step.cs:12]: Please rename to Tasks

Comment (dominikas) [reply FMM/Persistent/Step.cs:12]: done

File: FMM/Features/Step/StepController.cs (hunk 2 above)
Comment (dominikas) [reply FMM/Features/Step/StepController.cs:1]: we will do it tomorrow together :)

File: FMM/Features/Dream/DreamController.cs (hunk 1 above)
Comment (dominikas) [reply FMM/Features/Dream/DreamController.cs]: nice hint with reverting! thanks!

**Pull request merged**